tools/buildsteps/win32/buildlibdvd.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+MAKEFLAGS=""
+BGPROCESSFILE="$2"
+targetBuild="$3"
+bits="$4"
+tools="$5"
+
+[[ -f buildhelpers.sh ]] &&
+    source buildhelpers.sh
+
+do_git() {
+  local gitUrl="$1"
+  local gitFolder="$2"
+  cd "$LOCALBUILDDIR"
+  git clone -q "$gitUrl" "$gitFolder"
+  cd "$gitFolder"
+}
+
+do_autoget() {
+  local libname="$1"
+  cd "$LOCALBUILDDIR"
+  do_clean $libname $2
+  if [[ ! -d $libname ]]; then
+    do_git "https://github.com/xbmc/$libname.git" $libname
```

Comment:
this won't allow pulling different branches or specific commits.
best would be to use the same mechanism as for ffmpeg.